src/sentry/static/sentry/app/views/organizationIncidents/details/activity.jsx

```diff
@@ -312,6 +326,14 @@ class ActivityContainer extends React.Component {
 
 export default withApi(ActivityContainer);
 
+function removeFromArrayIndex(array, index) {
+  return [...array.slice(0, index), ...array.slice(index + 1)];
```

Comment:
Won't this not work for the last element in the list?

Reply:
hmm I think it does, but looking at it now we can just `splice` I think?

Reply:
Splice is good unless you need to keep the original array